packages/template-block/src/components/PreviewImage.tsx

```diff
@@ -93,7 +93,7 @@ export const PreviewImage = ({
                         ? blockAssets.previewCustom[0].height
                         : 'auto'
                 }
-                alt={blockSettings.altText || undefined}
+                alt={blockSettings.altText || 'Template preview'}
```

Comment:
`nit` nullish coalescing operator
```suggestion
                alt={blockSettings.altText ?? 'Template preview'}
```